src/Stan/Hie/Debug.hs

```diff
@@ -36,10 +37,17 @@ import Module (Module, ModuleName, moduleNameString, moduleStableString)
 import Name (Name, nameStableString)
 import Outputable (CodeStyle (CStyle), mkCodeStyle, printSDocLn)
 import Pretty (Mode (PageMode))
+import Text.Pretty.Simple (pPrint)
 import Var (ArgFlag (..))
 
 import qualified Text.Show
 
+
+debugHieFile :: FilePath -> [HieFile] -> IO ()
+debugHieFile path hieFiles = do
+    let mHieFile = find (\HieFile{..} -> hie_hs_file == path) hieFiles
+    whenJust mHieFile pPrint
```

Comment:
I've decided to add `pretty-simple` to our dependencies for now, since it's useful to debug files by names while we're still learning how to work with HIE files. Can remove this later to minimize dependencies...

Reply:
That's okay to keep it. It will improve the debugging workflow as well :+1: